script.js

```diff
@@ -0,0 +1,73 @@
+
+
+document.querySelector("#input").addEventListener("keydown", (event) => {
+  if (event.key == "Enter")
+    apiRequest();
+});
+
+document.querySelector("#search").addEventListener("click", () => {
+    apiRequest();
+});
+
+apiRequest = () => {
+
+  document.querySelector("#grid").textContent = "";
+
+  const url = 'https://api.unsplash.com/search/photos?query='+input.value+'&per_page=30&client_id=SouHY7Uul-OxoMl3LL3c0NkxUtjIrKwf3tsGk1JaiVo';
+
+  fetch(url)
+
+  .then(response => {
+    if (!response.ok) throw Error(response.statusText);
+      return response.json();

```

Comment:
```suggestion
    if (!response.ok) throw Error(response.statusText);
    
    return response.json();

```

before return we always add an empty line